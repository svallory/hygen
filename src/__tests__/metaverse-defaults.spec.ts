import { metaverse } from './metaverse-utils'
const enquirer = require('enquirer')

const failPrompt = () => {
  throw new Error('set up prompt in testing')
}

describe('metaverse', () => {
  beforeEach(() => {
    enquirer.prompt = failPrompt
  })

  metaverse('hygen-defaults', [['use-defaults']], { overwrite: true })
})

export { metaverse }
