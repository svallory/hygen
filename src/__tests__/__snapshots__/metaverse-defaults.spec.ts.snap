// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`metaverse hygen-defaults: use-defaults 1`] = `
Object {
  "actions": Array [
    Object {
      "status": "added",
      "subject": "given/hygen-defaults/default.md",
      "timing": -1,
      "type": "add",
    },
  ],
  "success": true,
  "time": 0,
}
`;
